docroot/themes/humsci/humsci_colorful/humsci_colorful.libraries.yml

```diff
@@ -0,0 +1,4 @@
+base:
+  css:
+    theme:
+      ../humsci_basic/dist/css/colorful.css: {}
```

Comment:
Libraries should only reference files within itself. It could define a dependency on a "colorful" library defined by `humsci_basic` but it should not directly target the file from outside itself.